packages/UI/src/components/Button/Button.tsx

```diff
@@ -1,116 +1,55 @@
-  const BB =
-    B.toString(16).length === 1 ? `0${B.toString(16)}` : B.toString(16);
-
-  return `#${RR}${GG}${BB}`;
-};
-
-const colorUnderButtons = (hexColor: string) => {
-  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexColor);
-  return result
-    ? `rgba(${parseInt(result[1], 16) - 102},${parseInt(result[2], 16) - 55},${
-        parseInt(result[3], 16) - 14
-      },1)`
-    : null;
-};
-
-const onHoverBackgroundColor = (hexColor: string) => {
-  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexColor);
-  return result
-    ? `rgb(${parseInt(result[1], 16) + 44},${parseInt(result[2], 16) + 44},${
-        parseInt(result[3], 16) + 44
-      })`
-    : null;
-};
-const Button = styled.a<ButtonProps>`
-  font-family: ${({ theme }) => theme.fonts.headline}, sans-serif;
-  background-color: ${({ theme, bgColor }) => bgColor || theme.colors.ACCENT_1};
-  color: ${({ theme, textColor }) => textColor || theme.colors.NEUTRAL_2};
-  transition: background-color 0.5s, color 0.5s;
-
-  width: ${({ width }) => `${width}em` || ''};
-  border: 0px;
-  border-bottom: 3px solid
-    ${({ theme, bgColor }) =>
-      colorUnderButtons(bgColor || theme.colors.ACCENT_1)};
+const Button = styled.a.attrs(({ disabled }) => ({
+  disabled,
+}))<ButtonProps>`
+  font-family: ${({ theme }) => theme.fonts.normal};
+  background-color: ${({ theme, buttonType }) => {
+    if (buttonType === 'primary') return theme.colors.ACCENT_5;
+    else if (buttonType === 'secondary') return theme.colors.ACCENT_8;
+    else if (buttonType === 'alternate') return theme.colors.MAIN_1;
+  }};
+  color: ${({ theme, buttonType }) => {
+    if (buttonType === 'primary') return theme.colors.MAIN_1;
+    else if (buttonType === 'secondary') return theme.colors.MAIN_1;
+    else if (buttonType === 'alternate') return theme.colors.MAIN_2;
+  }};
+  text-align: center;
+  text-transform: uppercase;
   cursor: pointer;
-  font-size: ${({ fontSize }) => `${fontSize}rem` || '1.5rem'};
-  margin-top: ${({ marginTop }) => `${marginTop}rem` || ''};
-  padding: 0.5rem;
-  padding-left: 1rem;
-  padding-right: 1rem;
-  border-radius: 16px;
-  font-weight: bolder;
-
-  ${({ hero }) =>
-    hero &&
-    css`
-      width: 80%;
-      margin: 0;
-      margin-top: 5%;
-      text-shadow: none;
-      line-height: 1.15;
-      display: inline-block;
-      text-align: center;
-      box-sizing: border-box;
-      font-stretch: normal;
-      font-variant-east-asian: normal;
-      font-variant-numeric: normal;
-      font-variant-caps: normal;
-      font-variant-ligatures: normal;
-      font-style: normal;
-    `}
-  ${({ modal }) =>
-    modal &&
-    css`
-      display: block;
-      width: 95%;
-      margin-top: 2%;
-      margin-left: auto;
-      margin-right: auto;
-    `}
-  ${({ intro }) =>
-    intro &&
-    css`
-      font-size: 2.5rem;
-      margin-top: 2%;
-      padding-left: 1.5rem;
-      padding-right: 1.5rem;
-    `}
+  font-size: 1rem;
```

Comment:
`font-family, font-size, font-weigh, line-height, text-transform` should all be removed from this file and we can import them from the typography atom that I made like this

```ts
import { typographyStyles } from '../Typography/Styled.Typography';
.
.
.
const Button = styled.button.attrs(({ disabled }) => ({
  disabled: disabled,
}))<ButtonProps>`
   ${typographyStyles['button']}
   ...rest of your styles
`
```